common/pom.xml

```diff
@@ -18,6 +18,18 @@
         <jackson.modules.version>2.8.5</jackson.modules.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
@wtritch answered my question.  I think something along the lines he has described above should be a comment within the pom.  The dependencies themselves don't need comments but the dependency management section is unlike any other spring dependency need we have in our applications.